client/src/components/fake-layout/Fake.css

```
@@ -0,0 +1,120 @@
+.parentContainer {
+    border: 1px dotted red;
+    height: 60px;
+    width: 100%;
+    font-size: 18px;
+    
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    align-items: top;
+}
+
+.order1 { order: 1 }
+.order2 { order: 2 }
+.order3 { order: 3 }
+.order4 { order: 4 }
+.order5 { order: 5 }
+
+.flexGrow132 { flex-basis: 32px; flex-grow: 1;}
+.flexGrow232 { flex-basis: 32px; flex-grow: 2;}
+.flexGrow332 { flex-basis: 32px; flex-grow: 3;}
+.flexGrow432 { flex-basis: 32px; flex-grow: 4;}
+.flexGrow532 { flex-basis: 32px; flex-grow: 5;}
+.flexGrow632 { flex-basis: 32px; flex-grow: 6;}
+.flexGrow732 { flex-basis: 32px; flex-grow: 7;}
+.flexGrow832 { flex-basis: 32px; flex-grow: 8;}
+.flexGrow932 { flex-basis: 32px; flex-grow: 9;}
+.flexGrow1032 { flex-basis: 32px; flex-grow: 10;}
```

I noticed that you use don't use some of these flex-grow values in the rest of the code. I also tried refactoring the `flex-basis: 32px` rule out into its own selector on a quick test page and it seemed to work. Is there a reason to specify the flex-basis in each of these rules?

Based on the way you're using it in the rest of your code, I'd suggest this: 
```suggestion
.flexBasis32 { flex-basis: 32px; }
.flexGrow1 {  flex-grow: 1; }
.flexGrow2 { flex-grow: 2; }
.flexGrow10 { flex-grow: 10; }
```